README.md

```diff
@@ -107,12 +107,89 @@ juju run-action etcd/0 snapshot target=/mnt/etcd-backups
 
 - **param** target: destination directory to save the resulting snapshot archive.
 
+
+### Restore
+
+Allows the operator to restore the data from a cluster-data snapshot. This
+comes with caveats and a very specific path to restore a cluster:
+
+The cluster must be in a state of only having a single member. So it's best to
+deploy a new cluster using the etcd charm, without adding any additional units.
+
+```
+juju deploy etcd new-etcd
+```
+
+> The above code snippet will deploy a single unit of etcd, as 'new-etcd'
+
+```
+juju run-action etcd/0 restore target=/mnt/etcd-backups
+```
+
+Once the restore action has completed, evaluate the cluster health. If the unit
+is healthy, you may resume scaling the application to meet your needs.
+
+- **param** target: destination directory to save the existing data.
+
+- **param** skip-backup: Don't backup any existing data.
+
+
+# Migrating etcd
+
+Migrating Etcd is a fairly easy task. The process is mostly copy/pasteable.
+
+Step 1: Snapshot your existing cluster. This is encapsuluted in the `snapshot`
+action.
+
+```
+$ juju run-action etcd/0 snapshot
+
+Action queued with id: b46d5d6f-5625-4320-8cda-b611c6ae580c
+```
+
+Step 2: Download the snapshot tarball from the unit that created the snapshot
```

Comment:
check the status of the action so you can grab the snapshot and verify the sum.

The copy.cmd result ouput is a copy/paste command for you to download the exact snapshot that you just created.